SPARCore/GADArticle.h

```diff
@@ -1,22 +1,26 @@
 #import <Foundation/Foundation.h>
+#import "GADRemoteModel.h"
 
-@interface GADArticle : NSObject
+@interface GADArticle : GADRemoteModel
 
+@property (nonatomic, strong) NSString *datePublished;
+@property (nonatomic, strong) NSString *brief;
+@property (nonatomic, strong) NSString *headerImage;
 @property (nonatomic, strong) NSString *publicationId;
+@property (nonatomic, strong) NSDate *dateEdited;
+@property (nonatomic, strong) NSString *articleId;
+@property (nonatomic, strong) NSString *title;
+@property (nonatomic, strong) NSString *content;
 @property (nonatomic, strong) NSArray *authors;
 @property (nonatomic, strong) NSURL *url;
 //@property (nonatomic, strong) NSInteger *pictureSize;
```

Comment:
I know you didn't add this in this commit, but could you remove this if you don't need it?